release/config.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data structures of toolchain configs."""
+
+import os
+
+
+class ConfigGroup(object):
+  """Group of configs that are generated by a single container.
+
+  Attributes:
+    distro: string, distro of container that generates the configs.
+    version: string, version of the configs.
+    container: string, the container registry entry of the container used to
+      generated the configs, e.g. gcr.io/cloud-marketplace/google/clang-ubuntu.
+    package: string, the Bazel package of the targets in this repo to generate
+      the configs.
+    platform_target: string, the platform target name of the corresponding RBE
```

Comment:
is this the just the name of the target or the full path to the target? (add an example?)